src/api/app/views/webui2/webui/kaminari/_paginator.html.haml

```diff
@@ -0,0 +1,12 @@
+= paginator.render do
+  %nav
+    %ul.pagination.justify-content-center
+      = first_page_tag unless current_page.first?
+      = prev_page_tag unless current_page.first?
+      - each_page do |page|
+        - if page.left_outer? || page.right_outer? || page.inside_window?
+          = page_tag page
+        - elsif !page.was_truncated?
+          = gap_tag
+      = next_page_tag unless current_page.last?
```

Comment:
```
- unless current_page.last?
  = next_page_tag
  = last_page_tag
```

Reply:
The same I wrote above:  The kaminari views are _generated_ and I don't see the point to change them to make them pass our lint tests.